docs/mkcloud.md

```diff
@@ -20,6 +20,14 @@ or testing purposes.
   $ sudo service libvirtd start  # to start the daemon
   ```
 
+  If you are using `libvirtd >= 1.3.0` then you should also start the
+  `virtlogd` service.
+
+  ```
+  $ sudo service virtlogd status # to check the status
+  $ sudo service virtlogd start  # to start the daemon
```

Comment:
There is this in the libvirt service unit

```
[Install]
WantedBy=multi-user.target
Also=virtlockd.socket
Also=virtlogd.socket
```

so my understanding is that if you `enable` the libvirt service file then virtlogd will also be enabled and both will be started on next reboot right?